stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/StreamVideoBuilder.kt

```diff
@@ -84,6 +86,7 @@ public class StreamVideoBuilder @JvmOverloads constructor(
     private val videoDomain: String = "video.stream-io-api.com",
     private val runForegroundServiceForCalls: Boolean = true,
     private val localSfuAddress: String? = null,
+    private val sounds: Sounds = Sounds(),
```

Comment:
How would a client app customize the sounds (and change other parameters), once the SDK is initialized?

Reply:
You can always reload the SDK with different parameters.